examples/qip-noisy-device-simulator.ipynb

```diff
@@ -10,11 +10,14 @@
     "\n",
```

Comment:
<p>Typos:</p><p> </p><ul><li>porjectQ ---&gt; ProjectQ</li><li>add space before (QIP)</li></ul><p> </p><p>You could break the first long paragraph in two. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qutip/qutip-notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I mean, just an if, that runs one block or the other depending on version

Reply:
I find there is already a space before (QIP)?